test/e2e/infra/vagrant/push_antrea.sh

```diff
@@ -181,10 +216,31 @@ if [ "$FLOW_AGGREGATOR" == "true" ]; then
             ssh -F ssh-config k8s-node-control-plane kubectl apply -f elk-flow-collector/elk-flow-collector.yml -n elk-flow-collector
             LOGSTASH_CLUSTER_IP=$(ssh -F ssh-config k8s-node-control-plane kubectl get -n elk-flow-collector svc logstash -o jsonpath='{.spec.clusterIP}')
             ELK_ADDR="${LOGSTASH_CLUSTER_IP}:4739:udp"
-
             $THIS_DIR/../../../../hack/generate-manifest-flow-aggregator.sh --mode dev -fc $ELK_ADDR > "${FLOW_AGG_YML}"
-        else
-            $THIS_DIR/../../../../hack/generate-manifest-flow-aggregator.sh --mode dev -fc $FLOW_COLLECTOR > "${FLOW_AGG_YML}"
+
+        elif [[ $FLOW_COLLECTOR == "GRAFANA" ]]; then
```

Comment:
pls update usage on L22 and include `GRAFANA`.
also a nit: technically it's clickhouse rather than grafana collecting the records. within fa configmap `clickHouse` is parallel to other `flowCollector`.

Reply:
I change the grafana to clickhouse.

Reply:
@dreamtalen I think we have different opinion here. Not sure whether I should put Grafana or clickhouse there?

Reply:
I got Shawn's point, for Flow Aggregator `clickhouse` is more accurate. 
`Grafana` is parallel to `ELK` in ` --flow-collector <Addr|ELK|clickhouse>` so it would be better in the argument of CLI IMO. Maybe we could get others' opinions too before make a decision here? @salv-orlando @heanlan

Reply:
> Grafana is parallel to ELK in --flow-collector <Addr|ELK>

+1. I also feel Grafana is more natural from the user perspective.